### ISSUE TITLE
There is no confirmation dialog

### ISSUE THREAD

**Issue by merlinuwe** [Canna71/obsidian-janitor#10]
I installed your plugin today and can get it run.

![image](https://user-images.githubusercontent.com/4138869/202871782-d801d9a2-4293-43ee-aeae-87b30b813a85.png)

But it does not end running ... and there comes no confimation dialog, even not after 1 hour.

The settings are set to default. 

What can I do?

Windows 10, lastest updates
Obsidian 1.0.3

**Comment by mikebarkmin** [comment]
For me, the error came from an invalid regex expression.

**Comment by Canna71** [comment]
@mikebarkmin could you elaborate? you mean the regex you can insert for excluding files or folders?

**Comment by mikebarkmin** [comment]
That's correct. My first attempt was `*.excalidraw`, but this is an invalid regular expression and lead to the error. `.*.excalidraw` is the correct one.

**Comment by Canna71** [comment]
I see, I could try and add a validator. Thank you for reporting this.

**Comment by Canna71** [comment]
Added a Regex Validator 1.0.7

**Comment by merlinuwe** [comment]
Thank you very much. I also had to delete a wrong excluded file (.xlsx) to get it work. Thanks to @mikebarkmin for the example and for you for implementing the validator.

Can you please give some regex examples for janitor excluded files in your documentation? (files in folder x younger/older than yyy-mm-dd, files that start or end with some letters, files with dates in their name, AND and OR, ...)

**Comment by Canna71** [comment]
Thank you both, I will try to improve documentation!

**Comment by MossabDiae** [comment]
Sorry for replying to an old issue but came here to manually exclude canvas files as they were deleted by Janitor.
The syntax is:  `.*.canvas`

@Canna71 can you kindly consider excluding them by default, I think most of the time they are used as orphans.
